xbmc/windowing/Resolution.h

```diff
@@ -62,23 +62,25 @@ struct EdgeInsets
   EdgeInsets(float l, float t, float r, float b);
 };
 
+//! @brief Provide info of a resolution
 struct RESOLUTION_INFO
 {
-  OVERSCAN Overscan;
+  OVERSCAN Overscan; //!< Screen overscan boundary
+  //!< Edge insets to scale the GUI to prevent the display notch from hiding a part of the GUI
   EdgeInsets guiInsets;
-  bool bFullScreen;
-  int iWidth;
-  int iHeight;
-  int iBlanking; /**< number of pixels of padding between stereoscopic frames */
-  int iScreenWidth;
-  int iScreenHeight;
-  int iSubtitles;
-  uint32_t dwFlags;
-  float fPixelRatio;
-  float fRefreshRate;
-  std::string strMode;
-  std::string strOutput;
-  std::string strId;
+  bool bFullScreen; //!< Specify if it is a fullscreen resolution, otherwise windowed
```

Comment:
Can you put the comment above each member and add whitespace between each member?

Reply:
it become a bit long but done